docker/docker_entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+# Ensure runner is available
+EXE_FILE_NAME=$(which $1)
```

Comment:
Maybe "command -v" is better than "which" as not all container images have the latter installed.